requirements.txt

```diff
@@ -65,6 +65,7 @@ sqlparse==0.3.0
 tablib==1.1.0
 Unidecode==1.1.1
 urllib3==1.25.8
+wagtail==2.8.0
```

Comment:
please make this latest 2.7.x release - 2.8 just came out so who knows what bugs are there with the new features. 2.7.x is safer.